src/components/organisms/AuthenticationModal/RegisterForm/RegisterForm.tsx

```diff
@@ -30,7 +30,7 @@ const RegisterForm: React.FunctionComponent<PropsType> = ({
     try {
       await signUp(data);
       afterUserIsLoggedIn && afterUserIsLoggedIn();
-      closeAuthenticationModal();
+      displaySaveCardForm();
```

Comment:
It feels weird to ask for the card data on register. Why not have the option just before paying?